src/editions/index.md

```diff
@@ -1,37 +1,36 @@
-other languages, but this also means that each update is smaller.
-
-After a while, all of those tiny changes add up. But, from release to release, it can be hard to look back
+Rust ships releases on a six-week cycle. This means that users get a constant
+stream of new features. This is much faster than updates for other languages,
+but this also means that each update is smaller.  After a while, all of those
+tiny changes add up. But, from release to release, it can be hard to look back
 and say *"Wow, between Rust 1.10 and Rust 1.20, Rust has changed a lot!"*
-Furthermore, since everything must be backwards compatible, we can't make certain
-tiny changes, like adding keywords because it might be already used as an
-identifier in existing code. Finally, you may sense that various parts
-of the project lags behind, since they update at different paces.
-If a new language feature lands in Rust 1.15, a new chapter about it may
-land in the book in 1.16, and the IDE integration may land in 1.17. This can feel
-like the pieces are not well coordinated, but if you compare Rust 1.15 to
-Rust 1.17, you'll see one cohesive feature being added.
-
-"Editions" are a concept to help with these kinds of problems. Every two or
-three years, we'll be producing a new edition of Rust. This serves different
-purposes for different people: for those working on Rust, it's a great way to
-focus on looking at the project as a whole, and being able to see the big
-picture. For those who haven't used Rust yet, or for those who may have tried
-Rust in the past and found it lacking in some way, a new edition signals that
-major advancements have landed, and you may want to give it a look!
+
+Every two or three years, we'll be producing a new *edition* of Rust. Each
+edition brings together the features that have landed into a clear package, with
+fully updated documentation and tooling. New editions ship through the usual
+release process.
+
+This serves different purposes for different people:
+
+- For active Rust users, it brings together incremental changes into an
+  easy-to-understand package.
+
+- For non-users, it signals that some major advancements have landed, which
+  might make Rust worth another look.
+
+- For those working *on* Rust, it provides a rallying point for the project as a
+  whole.
 
 ## Compatibility
 
-Editions are allowed to introduce minor kinds of incompatibilities, such as
-turning a warning into a hard error, or adding new keywords. At the same
-time, it's very important that we don't split the Rust ecosystem. Therefore,
-each project can list which edition that it's using, and the Rust compiler
-can link crates of any editions together. Therefore, if you're using Rust 2015,
-and one of your dependencies uses Rust 2018, it all works just fine. The
-opposite situation works as well.
+When a new edition becomes available in the compiler, crates must explicitly opt
+in to it to take full advantage. This opt in makes it possible to make
```

Comment:
... This opt-in, make it possible to make ...

Reply:
"Opting into the edition permits us to make backwards incompatible changes"